package.xml

```diff
@@ -14,6 +14,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <build_depend>rosdoc_lite</build_depend>
```

Comment:
This is not needed to build the tutorial examples.

Reply:
yes but it is needed for anyone who want's to build the documentation locally. I have changed the dependency to a `run_depend` as it is significantly cleaner than adding a manual `sudo apt install` command to the READMe